subprojects/docs/docs.gradle

```diff
@@ -404,11 +408,21 @@ samples {
 
         standalonePrecompiledScriptPlugins {
             sampleDirectory = samplesRoot.dir("build-organization/standalone-precompiled-plugins")
-            displayName = "Compose build logic using precompiled script plugins"
-            description = "Build and publish multiple precompiled script plugins."
+            displayName = "Publish precompiled script plugins"
+            description = "Compose build logic by building and publishing multiple precompiled script plugins."
+            category = "Build organization"
+            common {
+                from(templates.precompiledScriptPluginUtils)
+            }
+        }
+
+        multiProjectWithPrecompiledScriptPluginsInBuildSrc {
+            sampleDirectory = samplesRoot.dir("build-organization/multi-project-with-precompiled-script-plugins")
+            displayName = "Use precompiled script plugins from buildSrc"
+            description = "Configure a multi module project that composes build logic by building and using multiple precompiled script plugins."
```

Comment:
I am struggling with the display names and descriptions. I would like the display name to be very brief and elaborate in the description. Not sure if I got it right.